.github/workflows/label-reviews.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will set a number or reviewers depending on the labels
+name: Label Reviews
+# Trigger the workflow on pull requests
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+      - unlabeled
+  pull_request_review:
+    types:
+      - submitted
+      - edited
+      - dismissed
+jobs:
+  require-reviewers:
+    # Optional: skip check if no relevant label is present
+    # This needs to be kept in sync with the labels being checked
+    # These don't need to hold the entire label name and aren't case sensitive
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'Affects Balance') || contains(github.event.pull_request.labels.*.name, 'ongoing freeze') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label requires reviews
+        uses: BlueWeabo/label-requires-reviews-action@master
```

Comment:
Can you fork this repo into the GTNH org and refer to it here?

Reply:
Didn't know I had permissions to create repos here lol